### PR TITLE
[15.0][MIG] product_pricelist_revision: Migration to 15.0

### PR DESCRIPTION
**Pull request by CarlosRoca13**
cc @Tecnativa TT39030

please @carlosdauden @victoralmau review this

### CODE REVIEW

File: product_pricelist_revision/views/pricelist_view.xml [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">product.pricelist.item</field>
+        <field name="view_mode">tree,form</field>
+        <field
+            name="view_ids"
+            eval="
+            [
+                (5, 0, 0),
+                (0, 0, {'view_mode': 'form'}),
+                (0, 0, {'view_mode': 'tree', 'view_id': ref('product_pricelist_item_revision_tree_view')}),
```
Comment (victoralmau) [inline product_pricelist_revision/views/pricelist_view.xml]:
```suggestion
                (0, 0, {'view_mode': 'tree', 'view_id': ref('product_pricelist_item_revision_tree_view')}),
                (0, 0, {'view_mode': 'form'}),
```
Checking in runboat I could not access the listing mode (with this change, ok).

**Review by victoralmau — CHANGES_REQUESTED** [review]

**Comment by CarlosRoca13** [comment]
Change done @victoralmau

**Review by victoralmau — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]

**Pull request merged**